rust/src/participant.rs

```diff
@@ -296,7 +294,10 @@ mod tests {
                 .iter()
                 .map(|seed| {
                     (
-                        sign::PublicKey::from_slice(&randombytes(32)).unwrap(),
+                        // Or `UpdateParticipantPublicKey` ??? I don't
+                        // understand if this is supposed to be a
+                        // local seed dict or the global one.
+                        SumParticipantPublicKey::from_slice(&randombytes(32)).unwrap(),
```

Comment:
`UpdateParticipantPublicKey` instead of `SumParticipantPublicKey` is correct.

this is actually neither a `SeedDict` nor a `LocalSeedDict` but the inner dict of a `SeedDict`. basically the inner `collect()` creates a bunch of `HashMap<UpdateParticipantPublicKey, EncrMaskSeed>`, then the outer `collect()` creates the `SeedDict`. sorry for this monstrous double-collect :/